### PR TITLE
Add git autoupdate and licences into jquery-replacetext

### PR DESCRIPTION
**Pull request by LeaYeh**
**git repo url:** https://github.com/cowboy/jquery-replacetext

@maruilian11  could you help me check out the PR for #6200    ? thank you~
- [ ] Not found on cdnjs repo
- [ ] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Is this for #6166?

**Comment by PeterDaveHello** [comment]
Wrong title/commit message.

**Comment by LeaYeh** [comment]
@PeterDaveHello i added information about autoupdate and licences, title wrong?

**Comment by LeaYeh** [comment]
how about "Add git autoupdate and licences into jquery-replacetext" ?

**Comment by PeterDaveHello** [comment]
@LeaYeh a little bit better

**Comment by LeaYeh** [comment]
@PeterDaveHello okay i fix and rebase it.

File: ajax/libs/jquery-replace-text/package.json [hunk 1]
```diff
@@ -13,5 +13,17 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/cowboy/jquery-replacetext"
+  },
+  "licences": [
+    "MIT",
+    "GPL 2.0"
```
Comment (maruilian11) [inline ajax/libs/jquery-replace-text/package.json]: here should be `GPL-2.0`

**Comment by maruilian11** [comment]
@LeaYeh the PR's title also need to update.

**Comment by LeaYeh** [comment]
@maruilian11 i fix licence field typo error and change the title, cloud you check this again?

**Comment by LeaYeh** [comment]
oops i dont know why but the commit has 2 different hash number 7545a81 and 66732b6...

**Comment by maruilian11** [comment]
@LeaYeh but there is no tag on origin repo.

**Comment by LeaYeh** [comment]
@maruilian11 you are right.....i forgot to check it...

**Comment by LeaYeh** [comment]
okay... i open an [issue](https://github.com/cowboy/jquery-replacetext/issues/12) to ask that...
but the repo seems no one maintain it, maybe we can use another [fork](https://github.com/jsContribs/jquery-replacetext)

**Comment by PeterDaveHello** [comment]
Nope, you mentioned fork has no notable popularity, and it's a fork, not the origin one.

**Comment by PeterDaveHello** [comment]
oops ... this branch is too old, but didn't rebase with our latest master branch ...

**Pull request merged**